src/makielayout_integration.jl

```diff
@@ -40,6 +45,13 @@ function set_defaults!(attrs::Attributes)
     get!(attrs, :markersize, Observable(8px))
 end
 
+pkeys(aog) = aog.dict.vals[1].context.pkeys
```

Comment:
Maybe better to specify the type of `aog`, as this seems very specific. `aog.dict.vals[1]` should probably be `first(values(aog))`.  Also, it'd probably be better to introduce `pkeys(c::AbstractContext)`, because the data context has a `pkeys` field, but other contexts don't, so `pkeys(::AbstractContext) = nothing` should probably be the general fallback.

Reply:
Somehow, this feels a bit fragile, because `layoutplot!` accepts any `Algebraic`. Could you do it somewhere during the pipeline, where it is more clear what objects one is dealing with? For example, somewhere around here: https://github.com/JuliaPlots/AlgebraOfGraphics.jl/blob/master/src/makielayout_integration.jl#L49

Reply:
Actually never mind, this can probably stay as is, I'm working on a simplification of the system, so this will have to change in the near future anyways.